ObjectiveGit/GTIndex.h

```diff
@@ -202,7 +207,53 @@ NS_ASSUME_NONNULL_BEGIN
 ///
 /// Returns `YES` in the event of successful enumeration or no conflicts in the
 /// index, `NO` in case of error.
-- (BOOL)enumerateConflictedFilesWithError:(NSError **)error usingBlock:(void (^)(GTIndexEntry *ancestor, GTIndexEntry *ours, GTIndexEntry *theirs, BOOL *stop))block;
+- (BOOL)enumerateConflictedFilesWithError:(NSError **)error usingBlock:(void (^)(GTIndexEntry * _Nullable ancestor, GTIndexEntry * _Nullable ours, GTIndexEntry * _Nullable theirs, BOOL *stop))block;
+
+/// An enum for use with addPathspecs:flags:error:passingTest: below
+/// See index.h for documentation of each individual git_index_add_option_t flag.
+typedef NS_OPTIONS(NSInteger, GTIndexAddOptionFlags) {
```

Comment:
Nitpick: Maybe just `GTIndexAddOptions` ? That would make for more consistent wordings (e.g. "A combination of GTIndexAddOptionFlags flags" below).